images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -2,6 +2,8 @@ ARG DEBIAN_FLAVOR
 FROM oryxdevmcr.azurecr.io/private/oryx/githubrunners-buildpackdeps-${DEBIAN_FLAVOR} AS main
 ARG DEBIAN_FLAVOR
 ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR
+ARG AI_CONNECTION_STRING
```

Comment:
We should remove this and replace the `ARG AI_KEY` that's specified below on line 114 (since this is a multistage Docker image and this `ARG` _probably_ gets tossed in the next layer)

Reply:
That makes sense! Updated.